com.unity.render-pipelines.core/Editor/Lighting/GUIFramework/ClickAction.cs

```diff
@@ -0,0 +1,81 @@
+using System;
+using UnityEngine;
+
+namespace UnityEditor
+{
+    /// <summary>
+    /// Action for click
+    /// </summary>
+    public class ClickAction : HoveredControlAction
+    {
+        private int m_Button;
+        private bool m_UseEvent;
+        /// <summary>
+        /// Click count
+        /// </summary>
+        public int clickCount = 1;
+        /// <summary>
+        /// Action during onClick
+        /// </summary>
+        public Action<IGUIState, Control> onClick;
+        private int m_ClickCounter = 0;
+
+        /// <summary>
+        /// Initializes and returns an instance of ClickAction
+        /// </summary>
+        /// <param name="control">Current control</param>
+        /// <param name="button">Button ID</param>
+        /// <param name="useEvent">If use an event</param>
+        public ClickAction(Control control, int button, bool useEvent = true) : base(control)
+        {
+            m_Button = button;
+            m_UseEvent = useEvent;
+        }
+
+        /// <summary>
+        /// Get if the trigger condition is validated or not
+        /// </summary>
+        /// <param name="guiState">The gui state</param>
+        /// <returns></returns>
```

Comment:
Returns `true` if the trigger condition has been met. Otherwise, returns `false`.